shared-bindings/synthio/__init__.c

```diff
@@ -36,16 +37,111 @@
 #include "shared-bindings/synthio/MidiTrack.h"
 #include "shared-bindings/synthio/Synthesizer.h"
 
-//| """Support for MIDI synthesis"""
+//| """Support for multi-channel audio synthesis
 //|
-//| def from_file(file: typing.BinaryIO, *, sample_rate: int = 11025) -> MidiTrack:
+//| At least 2 simultaneous notes are supported.  samd5x, mimxrt10xx and rp2040 platforms support up to 12 notes.
+//|
+//| I'm a little teapot. I'm not on line 11, but I don't know what is.
+//| """
+//|
+//| class Envelope:
+//|     def __init__(
+//|         self,
+//|         attack_time: float,
+//|         decay_time: float,
+//|         release_time: float,
+//|         attack_level: float,
+//|         sustain_level: float,
```

Comment:
Should these be forced to be kwargs? Should they default to 0?

Reply:
I don't know what sensible defaults these would have, so I did not provide defaults

Reply:
without further re-work I'm not sure default values or kw-only values are possible, because I'm re-using the namedtuple constructor. I could change this at the cost of larger code size.

Reply:
I agree with having defaults in there, those @todbot listed are fine by me.

Reply:
OK, I'll use those as defaults. It's not as bad as I was worried it would be to allow default values. After the next update, it'll work like this:
```py
>>> synthio.Envelope()
Envelope(attack_time=0.1, decay_time=0.05, release_time=0.2, attack_level=1.0, sustain_level=0.8)
>>> synthio.Envelope(decay_time=7)
Envelope(attack_time=0.1, decay_time=7, release_time=0.2, attack_level=1.0, sustain_level=0.8)
>>> synthio.Envelope(7)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: extra positional arguments given
```